File: app/src/main/java/com/arthur/gloria/logired/features/login/registerclient/domain/model/ClientData.kt
package com.arthur.gloria.logired.features.login.registerclient.domain.model

import android.net.Uri

data class ClientData(
    val name: String,
    val lastname: String,
    val email: String,
    val numberPhone: String,
    val birthdate: String,
    val password: String,
    val confirmPassword: String = "",
    val imageUri: Uri? = null
)

File: lib/features/login/registerclient/domain/model/client_data.dart
class ClientData {
  final String name;
  final String lastname;
  final String email;
  final String numberPhone;
  final String birthdate;
  final String password;
  final String? imagePath;

  ClientData({
    required this.name,
    required this.lastname,
    required this.email,
    required this.numberPhone,
    required this.birthdate,
    required this.password,
    this.imagePath,
  });
}
